utils/python/CIME/XML/compilers.py

```diff
@@ -5,27 +5,40 @@
 from CIME.XML.standard_module_setup import *
 from CIME.XML.generic_xml import GenericXML
 from CIME.XML.files import Files
+from CIME.XML.compilerblock import CompilerBlock
```

Comment:
Supports both the old config_compilers.xml and the new one which was renamed from config_build.xml